deploy/packer/app.json

```diff
@@ -43,7 +43,7 @@
       "type":           "file",
       "direction":      "download",
       "source":          "/tmp/docker-worker.pub",
-      "destination":    "docker-worker-{{build_name}}-{{timestamp}}.pub"
+      "destination":    "docker-worker-{{build_name}}.pub"
```

Comment:
the purpose of this was because we were always generating a new private key, so we had to extra the public key from the AMI.  If we are reusing keys, we no longer need this at all.

Reply:
I removed it in a later commit, if you look at the file changes tab, you will see this code section removed.